app/views/layouts/application.html.erb

```diff
@@ -6,37 +6,39 @@
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
 
   <link href="https://cdn.earthdata.nasa.gov/eui/1.1.3/stylesheets/application.css" rel="stylesheet" />
-  <script src="https://d3js.org/d3.v3.min.js"></script>
-
+  
+  <%# See https://guides.rubyonrails.org/security.html#csrf-countermeasures %>
+  <%# See https://medium.com/rubyinside/a-deep-dive-into-csrf-protection-in-rails-19fa0a42c0ef %>
```

Comment:
seeing this again, I'm not sure we need these comments as reference b/c we can easily look it up, but I don't think this needs to change right now.